src/Bicep.LangServer.UnitTests/BicepExternalSourceRequestHandlerTests.cs

```diff
@@ -409,6 +405,7 @@ private Uri GetExternalSourceLinkUri(ExternalSourceLinkTestData testData, bool d
             SourceArchive? sourceArchive = entrypointUri is { } ?
                 SourceArchive.UnpackFromStream(SourceArchive.PackSourcesIntoStream(
```

Comment:
- Similar to other comments:  I recommend qualifying `documentLinks: null` to improve readability
- The qualifier Core.Workspaces here is not necessary and can be simplified